src/TGameDetails.h

```diff
@@ -444,6 +444,29 @@ qsl("<a href='https://abandonedrealms.com'>Website</a><br>"
                  "Busted. You can also use it as a starting point to create automated tests for your "
                  "own profiles!")},
 
+            {qsl("Accursed Lands"),
+             qsl("mud.accursed-lands.com"),
+             7979,
+             true,
+             qsl("<a href='https://www.accursed-lands.net'>https://www.accursed-lands.net</a><br>"
+                    "Discord<br>"
+                    "<a href='https://discord.gg/pUPdVra'>https://discord.gg/pUPdVra</a>"),
```

Comment:
I checked the invite and it is from ".splat" IIRC - but either my PC crashed before I could post this comment or I got sidetracked - but I hope this invite is a valid, unlimited uses/duration one - I'm not sure we need to show the actual URL as it can be review (I *think*) when it is hovered over...
```suggestion
                    "<a href='https://discord.gg/pUPdVra'>MUD's Official Discord guild</a>"),
```

Reply:
The game admins have checked this PR out and were happy with it, so I'd rather not change this on them. Also nobody calls discord servers guilds, that is not a name for the public 😅